eslint/core/deprecated-alias.ts

```diff
@@ -0,0 +1,159 @@
+import type {Rule} from 'eslint';
+import type ESTree from 'estree';
+
+const traverse = require('eslint-traverse');
```

Comment:
```suggestion
// eslint-disable-next-line @typescript-eslint/no-var-requires
const traverse = require('eslint-traverse');
```

We need to get rid of that package in future